server/controllers.js

```diff
@@ -0,0 +1,52 @@
+const models = require('./models');
```

Comment:
get, post and put requests don't actually handle errors. you will be sending back success status codes even if there is an error.